static/src/js/components/AutocompleteDisplay/AutocompleteDisplay.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Badge } from 'react-bootstrap'
+
+import { Button } from '../Button/Button'
+
+import './AutocompleteDisplay.scss'
+
+const AutocompleteDisplay = ({ selected, onRemoveAutocompleteValue }) => {
+  if (!selected.length) return null
+
+  return (
+    <li className="filter-stack-item">
```

Comment:
Wanted to note for discussion, I remember @trevorlang asking about this using `filter-stack-item` -- is this actually a filter stack item or does it just resemble it from an aesthetic standpoint? May need it's own class / styles.